src/qt/app.cpp

```diff
@@ -15,6 +15,11 @@
 #include <QtCore/QStringList>
 #include <QtWidgets/QApplication>
 
+#ifdef _WIN32
+    #include "wx/dynlib.h"
+    #include <Shlwapi.h>
```

Comment:
When cross-compiling the case can matter and, besides, we should be consistent with all the other includes of this header:
```suggestion
    #include <shlwapi.h>
```